crutest/src/main.rs

```diff
@@ -2185,11 +2351,149 @@ async fn perf_workload(
     }
     Ok(())
 }
+
+/// Prints a pleasant summary of the given region
+fn print_region_description(ri: &RegionInfo, encrypted: bool) {
+    println!("region info:");
+    println!("  block size:      {} bytes", ri.block_size);
+    println!("  blocks / extent: {}", ri.extent_size.value);
+    println!(
+        "  extent size:     {}",
+        human_bytes((ri.block_size * ri.extent_size.value) as f64)
+    );
+    println!(
+        "  extent count:    {}",
+        ri.total_blocks as u64 / ri.extent_size.value
+    );
+    println!("  total blocks:    {}", ri.total_blocks);
+    println!("  total size:      {}", human_bytes(ri.total_size as f64));
+    println!(
+        "  encryption:      {}",
+        if encrypted { "yes" } else { "no" }
+    );
+}
+
+async fn rand_read_write_workload(
+    guest: &Arc<Guest>,
+    ri: &mut RegionInfo,
+    cfg: RandReadWriteConfig,
+) -> Result<()> {
+    // Before we start, make sure the work queues are empty.
+    loop {
+        let wc = guest.query_work_queue().await?;
+        if wc.up_count + wc.ds_count == 0 {
+            break;
+        }
+        tokio::time::sleep(Duration::from_secs(2)).await;
+    }
+
+    let desc = match cfg.mode {
+        RandReadWriteMode::Read => "read",
+        RandReadWriteMode::Write => "write",
+    };
+
+    if cfg.fill {
+        println!("filling region before {desc}");
+        fill_workload(guest, ri, true).await?;
+    }
+
+    if cfg.blocks_per_io > ri.total_blocks {
+        bail!("too many blocks per IO; can't exceed {}", ri.total_blocks);
+    }
+
+    println!(
```

Comment:
I notice that often the upstairs `slog` messages can clobber this preamble and the
work you did to display the region info:
```
Transition from Reconcile to Active, client: 1, : downstairs, session_id: 04466381-d5e9-4b36-9cea-7d96cbd2de71
Mar 27 00:07:15.698 INFO Transition from Reconcile to Active, client: 2, : downstairs, session_id
-----------------------------------
: random write with 512 B chunks (1 block)
04466381-d5e9-4b36-9cea-7d96cbd2de71
Mar 27 00:07:15.698region info:
  block size:      512 bytes
   blocks / extent: 256
  extent size:     128 KiB
INFO  extent count:    2
  total blocks:    512
   total size:      256 KiB
All required reconciliation work is completed,   encryption:      no
session_id: 04466381-d5e9-4b36-9cea-7d96cbd2de71
Mar 27 00:07:15.698 INFO Set Downstairs and Upstairs active after reconciliatio, session_id: 04466381-d5e9-4b36-9cea-7d96cbd2de71
```

I had added at some point a wait for `guest.query_is_active()` to come back, but it seems like even that
is not waiting long enough to avoid `slog` clobbering things.  I'm loathe to suggest a sleep..

Reply:
I decided to mute the guest log by default for these new commands, with a `--verbose` CLI flag if you really want to see it; updated in 45c3c243df599036d239889a4e1d2dfa2e921174

(I left the existing commands unchanged, because I'm not sure if anyone is relying on their current behavior)

Reply:
I think being able to debugging the other tests when they fail in CI or elsewhere probably wants the
upstairs output, so that's good.  I may add a "log the upstairs to this file" option as well, as I really
don't need to see the upstairs logs 90% of the time.